k6 tests fails to run on Mac OS X

```
k6 (master) go1.12.9 test -race ./...
# github.com/loadimpact/k6/lib/netext/httpext [github.com/loadimpact/k6/lib/netext/httpext.test]
lib/netext/httpext/error_codes_test.go:104:12: undefined: syscall.EBFONT
lib/netext/httpext/error_codes_test.go:107:3: undefined: syscall.EBFONT
```

--- a/lib/netext/httpext/error_codes_test.go
+++ b/lib/netext/httpext/error_codes_test.go
@@ -101,10 +101,10 @@ func TestUnknownNetErrno(t *testing.T) {
 	var err = new(net.OpError)
 	err.Op = "write"
 	err.Net = "tcp"
-	err.Err = syscall.EBFONT // Highly unlikely to actually need to do anything with this error
+	err.Err = syscall.ENOTRECOVERABLE // Highly unlikely to actually need to do anything with this error
 	var expectedError = fmt.Sprintf(
 		"write: unknown errno `%d` on %s with message `%s`",
-		syscall.EBFONT, runtime.GOOS, err.Err)
+		syscall.ENOTRECOVERABLE, runtime.GOOS, err.Err)
 	var errorCode, errorMsg = errorCodeForError(err)
 	require.Equal(t, expectedError, errorMsg)
 	require.Equal(t, netUnknownErrnoErrorCode, errorCode)

